src/order/well_founded.lean

```diff
@@ -36,9 +36,6 @@ instance {α} [has_well_founded α] : _root_.is_asymm α has_well_founded.r :=
 theorem is_irrefl {α} {r : α → α → Prop} (h : well_founded r) : is_irrefl α r :=
 @is_asymm.is_irrefl α r (is_asymm h)
 
-instance {α} [has_well_founded α] : _root_.is_irrefl α has_well_founded.r :=
-is_asymm.is_irrefl
```

Comment:
I think I was wrong about this, the instances go in a weird direction

Reply:
I've added the instance back.